.github/workflows/release.yaml

```diff
@@ -163,7 +165,30 @@ jobs:
         id: get-version
         run: |
           ./radius/.github/scripts/release-get-version.sh ${{ steps.get-supported-versions.outputs.result }} radius
+      - name: Check if release branch exists
+        id: release-branch-exists
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const { data } = await github.rest.repos.getBranch({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              branch: '${{ steps.get-version.outputs.release-branch-name }}',
+            })
+
+            if (data.name == '${{ steps.get-version.outputs.release-branch-name }}') {
+              return 'true'
+            } else {
+              return 'false'
+            }
+      - name: Stop workflow if release branch exists and the event is a push to main
+        if: steps.release-branch-exists == 'true' && github.ref == 'refs/heads/main' && github.event_name == 'push'
+        run: |
+          echo "Release branch ${{ steps.get-version.outputs.release-branch-name }} already exists. Stop the workflow."
+          echo "IS_WORKFLOW_STOPPED=true" >> $GITHUB_ENV
```

Comment:
Instead of **IS_WORKFLOW_STOPPED**, should we have something more meaningful in the context of having another release branch already?

Reply:
what kind of thing are you thinking?

Reply:
I mean (and again please note that it is not that important) but it could be something like **ACTIVE_RELEASE_BRANCH**. `Workflow stopped` can mean something else. This is my opinion and would like to hear others' ideas too.

Reply:
looks like I can remove the env variable all together :) I updated the PR.